api/__tests__/handler.test.ts

```
@@ -12,26 +12,37 @@ interface ApiResponse {
 }));
 
-jest.mock('bcryptjs', () => ({
-  hash: jest.fn(),
-}));
+// Create mock functions
+const mockedCreateAccount = jest.fn();
+const mockedSendVerificationEmail = jest.fn();
+const mockedBcryptHash = jest.fn().mockResolvedValue('hashedPassword');
+
+// Assign mock implementations
+(createAccount as jest.Mock) = mockedCreateAccount;
```

Biome (1.9.4)

[error] The imported variable createAccount is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)